### ISSUE TITLE
 Output style not fixed with the same cluster_idx

### ISSUE THREAD

**Issue by yancheng31** [NVIDIA/pix2pixHD#71]
I have trained the instance-feature map. But when I run the inference, the output style is not fixed.
I find the style will change even I specify the same cluster_idx. 

Why the encode_features.py will generate different value of features_clustered*.npy even the input images are the same ?

How to specify a style to a cluster_idx ?

**Comment by tcwang0509** [comment]
How did you fix the style? By explicitly specifying a number in the `sample_features` function?

**Comment by yancheng31** [comment]
Yes, I specify a fix cluster_idx for a label in sample_features function.

For detail,  I have four labels, only one label that have two styles of training images, the others have one style for each label.  I expect the label=1's styles output can be control by cluster_idx=0/1. But the result shows when I change the cluster_idx, the label=1's styles could become label=2/3/4's style or any other styles on training image. 

How to solve the problem ?
Thanks.

**Comment by tcwang0509** [comment]
I don't really understand what you're saying. Basically if cluster_idx is fixed, the sampled feature is fixed, and if the label map is also fixed, the whole input to the network is fixed, so there's really no reason the output would be different. Can you confirm the input to the network is the same?
